app/assets/javascripts/application.js

```diff
@@ -13,8 +13,12 @@
 //= require jquery
 //= require jquery_ujs
 //= require jquery.ui.all
-//= require_tree .
+//= require ./do
+//= require ./jquery_extensions
+//= require ./ui
+//= require ./ox-account
```

Comment:
Does `ox-account` need to be required in this main JS manifest file?  My feeling (admittedly without a thorough review of everything else) is that since it is only loaded by another site (e.g. Tutor) it doesn't need to be included on normal Accounts page views.